src/components/tree/Tree.tsx

```diff
@@ -1,70 +1,144 @@
-import React, { FC } from "react";
-import { useParams } from "react-router";
-import { Tree as TreeCss, TreeSegment, Info } from "../StyleComponents";
-import Reference from "./Reference";
+import React, { FC, useState } from 'react';
 
-// TODO: Import this https://github.com/coreofscience/python-sap/blob/main/src/sap/template.html
-// REFERENCE FORMAT: https://github.com/coreofscience/python-sap/blob/main/src/sap/widget.py#L37-L97
+import CopyImage from '../vectors/CopyImage';
+import StarImgage from '../vectors/StarImage';
 
-import DATA from "./data.json";
+import Reference from './Reference';
+
+import './Tree.css';
+import DATA from './data.json';
+import src from '*.bmp';
```

Comment:
What does this do?

Reply:
Hi! How are you? I don't know I think you put that there

Reply:
I don't remember doing so, mind to delete it ?

Reply:
No! I already deleted it!

Reply:
Nice 👍